shaptools/shell.py

```diff
@@ -23,6 +23,10 @@ class ShellError(Exception):
     Exceptions in shell module
     """
 
+class FileDoesNotExistError(Exception):
```

Comment:
Add double new lines between class definitions. By definition:
- 2 new lines between methods outside classes
- 1 new line in method inside classes